### PR TITLE
Crossbow, speargun, and grenade updates

### PR DESCRIPTION
**Pull request by ghost**
- Ebow is now unlimited ammo again, and has the old name, but you need to cock it between each shot like the kinetic accelerator.
- You can now research the large crossbow after making it for an extra point in syndicate tech
- Added a kinetic speargun that has reusable 30 damage spears for hunting carps. Admin-spawnable only but will appear in summon guns.
- Saber (protolathe) SMG shoots as fast as c20r now because it was a little insane before.
- Fixed some misc shit with automatic guns having the wrong overlay when set to burst fire
- Fixed bulldog having an action button even though it's not burst fire anymore
- Grenades have an arm sound now that plays when you prime them (it's very subtle)

### CODE REVIEW

**Comment by WJohn** [comment]
I thought grenades already had a prime sound? When did this get removed?

**Comment by paprka** [comment]
No clue, but it sucks. There's no way to tell when someone's primed a grenade outside of them throwing it at you, and usually they do it halfway into the fuse so you have no time to react.

**Comment by dannno** [comment]
does that include chemnades
because the sparking thing they do currently is pretty dumb

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -276,7 +276,7 @@
 
 		add_fingerprint(usr)
 
-		if(!prevent_warning && !istype(W, /obj/item/weapon/gun/projectile/automatic/crossbow))
+		if(!prevent_warning && !istype(W, /obj/item/weapon/gun/energy/kinetic_accelerator/crossbow))
```
Comment (hornygranny) [inline code/game/objects/items/weapons/storage/storage.dm]: :(

Comment (paprka) [inline code/game/objects/items/weapons/storage/storage.dm]: What's the issue?

Comment (hornygranny) [inline code/game/objects/items/weapons/storage/storage.dm]: snowflaking a type in storage code, it should be an item variable instead imo

Comment (paprka) [inline code/game/objects/items/weapons/storage/storage.dm]: I didnt invent this feature for storage it was just a path replacement

But I agree and might change it when fixing the conflicts after steelpoint's ERT PR is merged (someone do that)

**Comment by paprka** [comment]
Conflicts fixed

**Pull request merged**